src/charm.py

```diff
@@ -51,6 +58,14 @@ def __init__(self, *args):
         # Gathers scrape job information from metrics endpoints
         self.metrics_consumer = MetricsEndpointConsumer(self)
 
+        # Exposes remote write endpoints
+        self.remote_write_provider = PrometheusRemoteWriteProvider(
+            self,
+            relation_name="receive-remote-write",
```

Comment:
Intentional mismatch with `DEFAULT_RELATION_NAME`?

Reply:
I don't think so. Fixed it.